model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -1,7 +1,5 @@
 package hu.blackbelt.judo.meta.jsl.validation
```

Comment:
Missing check for lambda, function that the parent model namespace have be "judo::functions" to avoid definitions in user's JSL.

Reply:
Done